packages/main/package.json

```diff
@@ -89,6 +69,7 @@
     "@electron-forge/maker-rpm": "^6.0.0-beta.43",
     "@electron-forge/maker-squirrel": "^6.0.0-beta.43",
     "@electron-forge/maker-zip": "^6.0.0-beta.43",
+    "@electron-forge/plugin-auto-unpack-natives": "^6.0.0-beta.43",
```

Comment:
Electron can load assets from a single combined file called an "asar". It is similar to a `.tar` file - many files are combined into one. The asar includes the apps javascript files for both the frontend and backend, node modules, html and css assets, etc. But apparently loading dependencies that use native code from an asar is inefficient. This plugin excludes just those dependencies from the asar, leaving them as separate files.